src/state/Maps/reducers.js

```diff
@@ -81,6 +223,51 @@ const setMapLayerStyleKey = (state, mapKey, layerKey, styleKey) => {
 	}
 };
 
+/**
+ * Set map layer option
+ * @param state {Object}
+ * @param mapKey {string}
+ * @param layerKey {string}
+ * @param optionKey {string}
+ * @param optionValue {*}
+ * @return {Object} state
+ */
+const setMapLayerOption = (state, mapKey, layerKey, optionKey, optionValue) => {
+	const layers = state.maps[mapKey]?.data?.layers;
```

Comment:
Should we check for mapKey, too?

Reply:
IMO we don't need it here. You are right that in some reducers is the check redundant